packages/boba/contracts/deploy/004-L2ERC20.deploy.ts

```diff
@@ -236,6 +244,20 @@ const deployFn: DeployFunction = async (hre) => {
     await registerL2LP.wait()
     console.log(`${token.name} was registered in LPs`)
   }
+
+  // Deploy xBoba
+  L2ERC20 = await Factory__xL2Boba.deploy('xBOBA Token', 'xBOBA', tokenDecimals)
```

Comment:
'tokenDecimals' here takes in the last token's decimals, can we make it fixed for xBOBA